### DISCUSSION TITLE
Error when exporting out of blender using latest GoB 3_6_3

### DISCUSSION THREAD

**Discussion by mattybisme** [JoseConseco/GoB#451] [General]
I'm also having an error when trying to export out of  blender to zbrush. I'm running on a Mac, and apple no longer supports OpenGL. Does this python script still leverage Open GL? The error message that spits out is similar to errors from other OGL addons. This script has been a godsend and would miss it greatly!(thanks for the development of it!) See attatched error:
<img width="822" alt="Screenshot 2023-11-20 at 8 37 12 PM" src="https://github.com/JoseConseco/GoB/assets/151474800/999a71a6-36d8-4b6a-b233-0fb3eea231d7">

**Comment by AlgoritmoDesign** [comment]
I have the same issue, somebody can help us?

**Comment by kromar** [comment]
the latest Gob 4.0.2 please check the releases section

**Comment by mattybisme** [comment]
works perfectly for me–thanks so much!
